tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -109,6 +112,54 @@ def run(self):
                 _cluster.current_job['status'] = "finished"
                 _cluster.is_managed = "yes"
                 _cluster.save()
+
+                # Creating a job to enable/disable profiling based on flag
+                # selected while import flow. This job is separate from
+                # import flow and not a child flow. This could be tracked
+                # independently once import if over.
+                if _cluster.volume_profiling_flag != "leave-as-is":
+                    # Possible  cluster.volume_profiling_state are enabled,
+                    # disabled and mixed
+                    # Possible cluster.volume_profiling_flag are enable,
+                    # disable and leave-as-is.
+                    # Below we compare the given above ( state and flag)
+                    if (_cluster.volume_profiling_state !=
+                            _cluster.volume_profiling_flag + "d"):
```

Comment:
i think we can modify this condition like "if _cluster.volume_profiling_flag in _cluster.volume_profiling_state" or create a map {"enabled": enable, "disabled": "disable"} and use the map here. appendind "d" and end i feel not good.

Reply:
I thought of doing the similar earlier, but then I realized that for just one comparison it would complicate the logic. I added the comments to understand what is happening. Is it fine ?

Reply:
Profiling enabling logic is blocking cluster-integration sync, It will take huge time for the cluster which has a large number of volumes. So by this time volumes details may be deleted by TTL.

Reply:
Profiling enabling logic is blocking cluster-integration sync, It will take huge time for the cluster which has a large number of volumes. So by this time volumes details may be deleted by TTL.

@shtripat @cloudbehl we need some logic to handle this here, otherwise no use of reducing import time.